test/fixtures/cdn-url/assets/global.css

```
@@ -0,0 +1,3 @@
+body {
+  font-family: 'MyCustom', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "MyCustom" (font-family-name-quotes)

(font-family-name-quotes)